packages/react/src/types.ts

```diff
@@ -17,3 +18,23 @@ export interface StyleProps {
   /** Pure css styles */
   style?: CSSProperties;
 }
+
+export interface RangeAnchor {
+  start: CalendarDate;
+  end: CalendarDate;
+}
+
+export interface DefinedRange {
+  key: string;
+  label: string;
+  rangeAnchor: RangeAnchor;
```

Comment:
can this be `value` instead of `rangeAnchor`?

Reply:
sure

Reply:
done